hack/README.md

```diff
@@ -131,12 +131,12 @@ Environment Variables and Supported Values
 
 | Env Variable       | Default       | Additional Values           |
 |--------------------|---------------|-----------------------------|
-|KUBEVIRT_PROVIDER   | k8s-1.16.2    | os-3.11.0-crio or okd-4.1   |
+|KUBEVIRT_PROVIDER   | k8s-1.16      | k8s-1.17, os-3.11.0-crio,   |
 |KUBEVIRT_STORAGE*   | none          | ceph, hpp                   |
```

Comment:
can you add nfs and ember_lvm to the storage?

Reply:
done